CVE-2022-3105.yml

```diff
@@ -361,8 +357,8 @@ interesting_commits:
       * Other commits that fixed a similar issue as this vulnerability
```

Comment:
I'd recommend adding a commit or two in the interesting commits section or at least mentioning why there weren't any (they were trivial, there weren't any at all, etc.)